client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -90,6 +97,25 @@ const HiddenSitesMessage = styled.div( {
 	marginBlockEnd: '1em',
 } );
 
+const ScrollButton = styled( Button, { shouldForwardProp: ( prop ) => prop !== 'visible' } )< {
+	visible: boolean;
+} >`
+	position: fixed;
+	display: flex;
+	opacity: ${ ( props ) => ( props.visible ? 1 : 0 ) };
+	align-items: center;
+	justify-content: center;
+	inset-block-end: 24px;
+	inset-inline-start: 24px;
+	z-index: 176;
```

Comment:
I wonder why this z index is necessary.

Reply:
without the z-index the scroll to top button gets hidden behind sites thumbnail when in mobile view, So I adapt the order of the z-index from the floating help button. 
![image](https://user-images.githubusercontent.com/47489215/187554436-585b1a66-97ef-49ef-a767-cf6bcb50bd91.png)

Reply:
I think it's safe to remove the `z-index` property from the `.site-thumbnail-icon` class, right? Removing it yielded no difference for me 🤔

Reply:
I think the z-index only become apparent when the thumbnail loader is shown so the icon basically sits on top of the loader

Reply:
Can't we apply that z-index to the loader itself, then?

Reply:
yeah I've removed the unnecessary z-index